assemblies/psw-ce/src/main/resources-filtered/workbench.sh

```diff
@@ -74,7 +74,7 @@ setPentahoEnv
 JAVA_LOCALE_COMPAT=""
 if $($_PENTAHO_JAVA -version 2>&1 | grep "version \"11\..*" > /dev/null )
 then
-  JAVA_LOCALE_COMPAT="-Djava.locale.providers=COMPAT,SPI"
+  JAVA_LOCALE_COMPAT="--add-opens java.base/java.net=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/sun.net.www.protocol.jar=ALL-UNNAMED -Djava.locale.providers=COMPAT,SPI"
```

Comment:
While this keeps java 11 changes fairly contained, I'd prefer this to be its own variable. I expect us to keep the java.locale.providers indefinitely, but I'm assuming these add-opens will be short lived. Giving this it's own name will make it more clear what it is for, and easier to remove later.